_docs/changelog.md

```diff
@@ -9,6 +9,10 @@ permalink: /docs/changelog/
 Current version of the API is `1.9.0`
 Current version of the SDKs are `1.9.0`
 
+new changes
```

Comment:
Here we should have the new version number. Please coordinate with @cesarecamurani for the public release and define the new number.

Reply:
I know it, but because I didn't know what version it will be I have decided to put this "mock"